### PR TITLE
Shop template revision

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | n/a
| Versions      | 3.x, master

### CODE REVIEW

File: docs/guide/shop_templates/shop_twig_functions.md [hunk 1]
```diff
@@ -1,126 +1,69 @@
...
 ## Functions
 
 |Name|Parameters|Description|
 |--- |--- |--- |
-|`ses_render_field`|`CatalogElement $catalogElement`</br>`string $fieldIdentifier = ''`</br>`array $params = array()`|Renders a CatalogElement containing a Field|
-|`ses_navigation`|`array $params = array()`|Returns navigation node tree|
+|`ses_render_field`|`CatalogElement $catalogElement`</br>`string $fieldIdentifier = ''`</br>`array $params = array()`|Renders a Field of a CatalogElement|
 |`ses_product`|`array $params = array()`|Returns a product node</br>`{% set product = ses_product({'sku': line.sku, 'variantCode': line.variantCode }) %}`|
 |`ses_variant_product_by_sku`|`string $sku`|Fetches a VariantProductNode for the given SKU|
+ses_assets_by_group
```
Comment (dabrt) [inline docs/guide/shop_templates/shop_twig_functions.md:10]: Should this function name be followed by an example and a description?

File: docs/guide/shop_templates/shop_twig_functions.md [hunk 2]
```diff
@@ -1,126 +1,69 @@
...
+|`ses_total_comparison`|-|Gets total basket lines array for the basket flyout|
 |`ses_check_product_in_comparison`|`$sku`</br>`$variantCode = null`|Returns true if product with the given SKU and variant code is already in the product comparison|
 |`ses_check_product_in_wish_list`|`$sku`</br>`$variantCode = null`|Returns true if product with the given SKU and variant code is already in the wishList|
 |`ses_pagination`|`Pagerfanta $pagerfanta`</br>`$viewName = null`</br>`array $options = array()`|Renders pagination|
 |`get_stored_baskets`|-|Returns stored baskets for the current user|
-|`ses_user_menu`|-|Returns rendered user menu</br>Combines parts of all configured bundles|
-|`ses_track_basket`|`Basket $basket`</br>`$mode`</br>`array $params = array()`|Returns a list with rendered template snippets for basket tracking|
-|`ses_track_product`|`ProductNode $catalogElement`</br>`$mode`</br>`array $params = array()`|Returns a list with rendered template snippets for product tracking|
-|`ses_track_base`|`array $params = array()`|Returns a list with rendered template snippets for general tracking|
+|`ses_user_menu`|-|Returns rendered user menu|
```
Comment (dabrt) [inline docs/guide/shop_templates/shop_twig_functions.md:23]:
```suggestion
|`ses_user_menu`|-|Returns the rendered user menu|
```

**Review by dabrt — APPROVED** [review]

File: docs/guide/shop_templates/shop_twig_functions.md (hunk 1 above)
Comment (DominikaK) [inline docs/guide/shop_templates/shop_twig_functions.md:10]:
```suggestion
```

Comment (DominikaK) [reply docs/guide/shop_templates/shop_twig_functions.md:10]: Unfortunate leftover from my notes ;)

**Pull request merged**